src/lib/biopam.ml

```diff
@@ -0,0 +1,93 @@
+(*
+Provide tools via Biopam: https://github.com/solvuu/biopam
+*)
+
+open Common
+open Run_environment
+
+let rm_path = Workflow_utilities.Remove.path_on_host
+
+module Opam = struct
+
+  let dir ~meta_playground = meta_playground // "opam_dir"
+  let bin ~meta_playground = dir ~meta_playground // "opam"
+  let root ~meta_playground = dir ~meta_playground // ".opam"
+
+  (* TODO:
+     Instead of just making sure that this file exists? Wouldn't it be better
+     to make sure that a command from this program gives the right output?
+     ie. $ opam --version = 1.2.2 *)
+  let target ~host ~meta_playground =
+    KEDSL.single_file ~host (bin ~meta_playground)
+
+  let com ?(switch=true) ~meta_playground fmt =
+    let bin = bin ~meta_playground in
+    let root = root ~meta_playground in
+    (* Pass the ROOT and SWITCH args as environments to not disrupt the flow of
+       the rest of the arguments. *)
+    if switch then
+      Printf.sprintf ("OPAMROOT=%s OPAMSWITCH=0.0.0 %s " ^^ fmt) root bin
+    else
+      Printf.sprintf ("OPAMROOT=%s %s " ^^ fmt) root bin
+
+  let file ?(switch=true) ~meta_playground ~package f =
+    let s = com ~switch ~meta_playground "config var %s:lib" package in
+    (* Are there places where this tick logic is flaky? *)
+    (Printf.sprintf "`%s`" s) // f
```

Comment:
Maybe this should be called "opam_file_command" ?

Something called `file` will make us use `Filename.quote` which would break the backquotes thing.